### PR TITLE
Add script to move G7 communications

### PR DESCRIPTION
**Pull request by robyoung**
Move the G7 communications files to the new bucket while adding the `last_modified` date as a timestamp.

### CODE REVIEW

File: scripts/oneoff/move-communications.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+sys.path.insert(0, '.')
+
+import re
+from docopt import docopt
+
+from dmutils.s3 import S3
+
+
+def make_coppier(src_bucket, target_bucket):
+    def copy_file(src_path, target_path):
```
Comment (pcraig3) [inline scripts/oneoff/move-communications.py]: This is super cool.  Didn't know you could do this.

<sub>p.s. copier</sub>

**Comment by pcraig3** [comment]
:+1:

File: scripts/oneoff/move-communications.py (hunk 1 above)
Comment (carolinegreen) [inline scripts/oneoff/move-communications.py]: Is the spelling intentional?

Comment (robyoung) [inline scripts/oneoff/move-communications.py]: No, I just can't words.

**Pull request merged**